TTYC.Application/Stores/GetStore/GetStoreHandler.cs

```diff
@@ -17,7 +17,7 @@ public GetStoreHandler(ApplicationDbContext dbContext)
         public async Task<Store> Handle(GetStoreQuery query, CancellationToken cancellationToken)
         {
             var store = await dbContext.Stores
-                .Include(x => x.Products)
+                .Include(x => x.Products.Where(x => x.IsActive == true))
```

Comment:
.Where(x => x.IsActive)
Are you sure this works?

Reply:
Yes